scripts/buildprbody.py

```diff
@@ -17,88 +17,112 @@
 import pandas as pd
 import pathlib
 
-class ChangeType(IntEnum):
-    UNKNOWN = 0
-    DELETED = 1
-    ADDED = 2
-    CHANGED = 3
-
-
-def get_commit_link(name):
-    """Return a string with a link to the last commit for the given
-        API Name.
-    args:
-        name (str): The name of the api.
-    """
+from changesummary import ChangeType
+
+SCRIPTS_DIR = pathlib.Path(__file__).parent.resolve()
+CHANGE_SUMMARY_DIR = SCRIPTS_DIR / "temp"
 
-    url = "https://github.com/googleapis/google-api-python-client/commit/"
-    sha = None
-    api_link = ""
 
-    file_path = pathlib.Path(directory).joinpath("{0}.sha".format(name))
-    if file_path.is_file():
-        with open(file_path, "r") as f:
-            sha = f.readline().rstrip()
-            if sha:
-                api_link = "[{0}]({1}{2})".format(" [More details]", url, sha)
+class BuildPrBody:
+    """Represents the PR body which contains the change summary between 2
+    directories containing artifacts.
+    """
 
-    return api_link
+    def __init__(self, change_summary_directory):
+        """Initializes an instance of a BuildPrBody.
+
+        Args:
+            change_summary_directory (str): The relative path to the directory
+            which contains the change summary output.
+        """
+        self._change_summary_directory = change_summary_directory
+
+    def get_commit_uri(self, name):
+        """Return a uri to the last commit for the given API Name.
+        args:
+            name (str): The name of the api.
+        """
+
+        url = "https://github.com/googleapis/google-api-python-client/commit/"
+        sha = None
+        api_link = ""
+
+        file_path = pathlib.Path(self._change_summary_directory).joinpath(
```

Comment:
I would prefer the new code chooses a consistent way to join paths. I see below you use `/`

Reply:
Done.